client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-fallback-credentials/test/index.tsx

```diff
@@ -0,0 +1,261 @@
+/**
+ * @jest-environment jsdom
+ */
+import { waitFor, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import nock from 'nock';
+import React from 'react';
+import wpcomRequest from 'wpcom-proxy-request';
+import { useSiteSlugParam } from 'calypso/landing/stepper/hooks/use-site-slug-param';
+import wp from 'calypso/lib/wp';
+import SiteMigrationCredentials from '..';
+import { StepProps } from '../../../types';
+import { RenderStepOptions, mockStepProps, renderStep } from '../../test/helpers';
+
+jest.mock( 'calypso/lib/wp', () => ( {
+	req: {
+		get: jest.fn(),
+		post: jest.fn(),
+	},
+} ) );
+
+jest.mock( 'wpcom-proxy-request', () => jest.fn() );
+jest.mock( 'calypso/landing/stepper/hooks/use-site-slug-param' );
+
+( useSiteSlugParam as jest.Mock ).mockImplementation( () => 'site-url.wordpress.com' );
+
+// jest.mock( 'calypso/landing/stepper/hooks/use-query', () => ( {
```

Comment:
can we remove this?

Reply:
Thank you for catching that! Removed now.